source/linux/Overview/_Processor_SDK_Linux_Software_Developers_Guide.rst

```diff
@@ -111,7 +111,7 @@ suggestions at `E2E <https://e2e.ti.com/>`__.
 
 .. ifconfig:: CONFIG_sdk in ('SITARA')
 
-    .. ifconfig:: CONFIG_part_variant not in ('AM62X', 'AM62AX', 'AM62PX')
+    .. ifconfig:: CONFIG_part_variant not in ('AM62X', 'AM62AX', 'AM62PX', 'AM335X', 'AM437X', 'AM65X')
```

Comment:
@jeevantelukula You missed removing these sections from individual device toc tree? 

For Instance - https://software-dl.ti.com/processor-sdk-linux/esd/AM65X/09_03_05_02/exports/docs/linux/Industrial_Protocols.html shouldn't be visible in the device level toc tree